README.md

```diff
@@ -135,11 +135,17 @@ Options:
                                   hg|\.mypy_cache|\.nox|\.tox|\.venv|\.svn|_bu
                                   ild|buck-out|build|dist)/]
 
+  --extend-exclude TEXT           Like --exclude, but adds additional files
+                                  and directories on top of the excluded
+                                  ones. (Useful if you simply want to add to
+                                  the default)
```

Comment:
```suggestion
                                  ones (useful if you simply want to add to
                                  the default).
```

This style has some history, but it honestly really doesn't matter. British and American English are both used with no consistency already :p
```
  --pyi                           Format all input files like typing stubs
                                  regardless of file extension (useful when
                                  piping source on standard input).
```

*don't forget update the docs/installation_and_usage.md document too if you decide this is okay

Reply:
Just curious, do you mean this formatting change is a British/American difference?

Reply:
I poorly wrote that comment (as I always do unfortunately). I meant that consistency in our docs is already a bit hazy, using the mixture British and American English in our docs as an example. And that this comment is effectively a nit.